### PR TITLE
bpo-33876: Mention relevant pythonesque implementations

### PR DESCRIPTION
**Pull request by andresdelfino**
https://bugs.python.org/issue33876

### CODE REVIEW

**Comment by willingc** [comment]
Hi @andresdelfino, Thanks. If we are mentioning MicroPython, we should also mention CircuitPython as well.

**Review by willingc — COMMENTED** [review]
Looks good. Let's add CircuitPython too.

**Comment by andresdelfino** [comment]
CircuitPython added! I believe I have highlighted the main differences right. Wasn't sure about the URL, but https://learn.adafruit.com/welcome-to-circuitpython/what-is-circuitpython looked somewhat weird. The GitHub repository seemed better.

File: Doc/reference/introduction.rst [hunk 1]
```diff
@@ -83,6 +83,11 @@ MicroPython
    possible to allow the transfer of code with ease from the desktop.  For more
    information, see `the MicroPython website <https://micropython.org/>`_.
 
+CircuitPython
+   A MicroPython fork targeted towards students and beginners, with a strong
+   focus on Adafruit boards.  For more information, see `the CircuitPython
```
Comment (willingc) [inline Doc/reference/introduction.rst]: Thanks @andresdelfino. All of Adafruit's boards are open-hardware which would be good to mention.  Perhaps "...on Adafruit's open-hardware boards.

**Review by willingc — APPROVED** [review]

File: Doc/reference/introduction.rst (hunk 1 above)
Comment (andresdelfino) [reply Doc/reference/introduction.rst]: Done :)

**Review by willingc — APPROVED** [review]
Thanks @andresdelfino. I'll leave this open for another day for others to comment, but this looks good to me. Thanks for the helpful docs work that you are doing 👍

**Comment by andresdelfino** [comment]
Please **do not merge**, see issue.

**Comment by Mariatta** [comment]
Thanks for the PR, but as I mentioned in bpo, I just don't think this list is needed in our documentation.

**Pull request closed without merging**